Packages/com.unity.inputsystem/Documentation~/Actions.md

```diff
@@ -29,11 +29,11 @@ Actions also make it simpler to create a system that lets your players [customiz
 
 ## Overview
 
-When scripting with Actions in the Input System, there are number of important classes you can use, which are described here:
+When scripting with Actions in the Input System, there are number of important API you can use, which are described here:
 
-|Class|Description|
+|API name|Description|
 |-----|-----------|
-|[`InputSystem.actions`](../api/UnityEngine.InputSystem.InputSystem.html)|A reference to the set of actions configured in the [Input Actions editor](ActionsEditor.md). |
+|[`InputSystem.actions`](../api/UnityEngine.InputSystem.InputSystem.html)|A reference to the set of actions assigned as the [project-wide Actions](./Processors.md).|
```

Comment:
That link looks potentially wrong - should be project wide actions - not processors

Reply:
ProjectWideActions.md

Reply:
fixed in https://github.com/Unity-Technologies/InputSystem/pull/1867/commits/0c5b32bba250166b2b85754d06e03c25b2d3b6fd

Reply:
I see update fixes this